### PR TITLE
Add deployment config file for one click deployment.

### PR DESCRIPTION
**Pull request by praveenkumarct**
Added deployment config for the Adyen Integration project to support one-click-deployment.

**NOTE: Don't merge this PR. This is for testing purpose.**

### CODE REVIEW

File: deployment/gcp/deploy.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "applicationType": "service",
+    "dockerFilePath": "extension/",
+    "name": "Extension",
+    "properties": {}
```
Comment (lojzatran) [inline deployment/gcp/deploy.json:6]: Remove this empty properties.

**Review by lojzatran — CHANGES_REQUESTED** [review]
Location does not seem correct to me. Maybe put it in the root folder of the whole repo?

File: deployment/gcp/deploy.json (hunk 1 above)
Comment (praveenkumarct) [reply deployment/gcp/deploy.json]: https://github.com/commercetools/commercetools-adyen-integration/pull/1005/commits/8bd218908ba6fae9ac50347683860496daed662e

**Comment by lojzatran** [comment]
Btw let's leave this file now in this PR and not merge it to main branch yet. We might do more updates once we have the test implementation.

**Comment by praveenkumarct** [comment]
>Location does not seem correct to me. Maybe put it in the root folder of the whole repo?

As mentioned in the JIRA ticket, we discussed this location in the initial discussions.
This location doesn't matter with the location we are providing for DockerFilePath. DockerFilePath property should be considered from the main folder of the project.

**Comment by lojzatran** [comment]
> > Location does not seem correct to me. Maybe put it in the root folder of the whole repo?
> 
> As mentioned in the JIRA ticket, we discussed this location in the initial discussions. This location doesn't matter with the location we are providing for DockerFilePath. DockerFilePath property should be considered from the main folder of the project.

Which JIRA ticket are you referring to? Can you send it in the public slack channel for this task? The deploy.json file is located here: `deployment/gcp/deploy.json`. Didn't we agree that integrator will not care if it's GCP or AWS or Azure?

**Comment by praveenkumarct** [comment]
https://github.com/commercetools/commercetools-adyen-integration/pull/1005/commits/2bc390f57fe91b1ae3ef4852b9a6381c91dceef3
As discussed in the slack, I made changes. 
https://commercetools.slack.com/archives/C03CYLEDH9Q/p1661533921826009

 We will not merge this PR for now and use this branch for testing Integrations.

File: connect.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "deployAs":[
```
Comment (butenkor) [inline connect.json:2]: `deployments` might be better wording

File: connect.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    {
+      "applicationType": "service",
+      "dockerFilePath": "extension/",
```
Comment (butenkor) [inline connect.json:5]: are slashes required in the end?

**Comment by merijndebeer-adyen** [comment]
Thank you for your contribution. We have reviewed this pull request and decided to not merge it since it does not fit in the current version of the integration. Regards Adyen Product Management

**Pull request closed without merging**